vshard.go

```diff
@@ -232,6 +230,10 @@ func prepareCfg(cfg Config) (Config, error) {
 		return Config{}, fmt.Errorf("%v: %v", ErrInvalidConfig, err)
 	}
 
+	if cfg.DiscoveryTimeout == 0 {
+		cfg.DiscoveryTimeout = 1 * time.Minute
```

Comment:
it is worth transferring the default value ​​to a constant

Reply:
Done